deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -5,19 +5,19 @@ metadata:
     alm-examples: |-
       [
         {
-          "apiVersion": "apps.openshift.io/v1alpha1",
```

Comment:
Is the version going to remain as `0.0.23`?

Reply:
This, I think is the template, but the version probably isn't used anywherre.